src/Logging/TeamCity/TeamCityLogger.php

```diff
@@ -181,6 +183,45 @@ public function testSkipped(Skipped $event): void
         $this->writeMessage('testIgnored', $parameters);
     }
 
+    /**
+     * @throws InvalidArgumentException
+     */
+    public function testSuiteSkipped(TestSuiteSkipped $event): void
```

Comment:
Looks like we are missing a test for `markTestSkipped()` being called in a `setUpBeforeClass()` method, for example.

Reply:
I will take care of this.